packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.2
+
+* Handle `BehindLiveWindowException`
```

Comment:
Please fix this to follow the CHANGELOG style guide that was linked in the PR checklist. You've checked the box, but aren't actually following the style described there.